addons/mail/models/mail_tracking_value.py

```diff
@@ -37,15 +36,26 @@ class MailTracking(models.Model):
     tracking_sequence = fields.Integer('Tracking field sequence', readonly=1, default=100)
 
     def _compute_field_groups(self):
+        if any(tracking.field.groups for tracking in self):
+            all_groups = self.env['ir.model.data'].search([('model', '=', 'res.groups')])
+            groups = {group.res_id: group.complete_name for group in all_groups}
+
         for tracking in self:
-            model = self.env[tracking.mail_message_id.model]
-            field = model._fields.get(tracking.field)
-            tracking.field_groups = field.groups if field else 'base.group_system'
+            if tracking.field.groups:
+                ids = [groups[x] for x in tracking.field.groups.ids]
```

Comment:
Does this work ? It seems the `groups` field of `ir.model.fields` is not implemented oO
```sql 
select COUNT(*) from ir_model_fields_group_rel;
0
```
The table is empty :/